subsys/bluetooth/host/iso.c

```diff
@@ -875,6 +877,9 @@ void hci_le_cis_established(struct net_buf *buf)
 	}
 
 	if (!evt->status) {
+		/* Reset sequence number */
+		iso->iso.seq_num = 0;
```

Comment:
If I understand this correctly, the seq_num is reset at the start of the connection/transfer for unicast (`le_cis_established()`), but at the end for broadcast (`big_complete()`).
If this is a correct understanding - is there a reason for this difference?

Reply:
`big_complete` is the start of a BIG :)